### ISSUE TITLE
Touchpad issues on Linux

### ISSUE THREAD

**Issue by linux-man** [boppreh/mouse#103]
I'm on a laptop, and found that the touchpad move event are associated with EV_ABS, so to receive "MoveEvent" I had to change `listen(queue)` at _nixmouse.py:

```python
elif type == EV_REL:
```
to
```python
elif type in (EV_REL, EV_ABS):
```
The coordinates look fine.

What looks difficult is to detect which button was pressed (all clicks are "LEFT"), 2-fingers scroll and discriminate the "moving" touch from the "click" touch.
Can that info be hidden on EV_MSC events?

**Comment by linux-man** [comment]
Not so easy: with that change I got a "Segmentation fault (core dumped)" when calling a Gtk.menu with right click in another class. Weird.

**Comment by changeso** [comment]
are you deal with the issue? I have the same problem, my process only receive EV_ABS, I cann't use EV_REL to get input_event

**Comment by linux-man** [comment]
No. That's why I decided to use different libraries, the other being pyinput. Have a look at https://github.com/linux-man/inmon.
